src/pages/Account/Login.tsx

```diff
@@ -26,6 +33,39 @@ export const Login: FC<LoginProps> = ({ formType = "initial" }) => {
 
   const redirectAfterLogin = () => {};
 
+  // If there is a customToken param in the URL, use that to login
+  const firebase = useFirebase();
+  const {
+    replace: replaceUrlUsingRouter,
+    location: { search },
+  } = useHistory();
+  const { loading: isCustomTokenLoginLoading } = useAsync(async () => {
+    const customToken = new URLSearchParams(search).get("customToken");
+
+    if (!customToken) return;
+
+    // @debt: move this into api/auth or similar?
+    await firebase.auth().signInWithCustomToken(customToken);
+    // .catch((error) => {
+    //   // TODO: Handle this with Bugsnag or similar?
+    //   console.error(error);
+    //
+    //   throw error;
+    // });
+
+    // Remove customToken from the url params
+    const currentUrlWithoutCustomToken = new URL(window.location.pathname);
+    currentUrlWithoutCustomToken.search = window.location.search;
+    currentUrlWithoutCustomToken.searchParams.delete("customToken");
+    // @debt This should work, but it doesn't seem to be properly updating the URL for some reason. Bug with react-router's replace method?
+    replaceUrlUsingRouter(currentUrlWithoutCustomToken.toString());
+
+    // @debt I feel like it would be better to explicitly redirect somewhere once we have logged in.. but our other signIn
+    //  code in LoginForm doesn't seem to..
+  }, [firebase, replaceUrlUsingRouter, search]);
+
+  if (isCustomTokenLoginLoading) return <LoadingPage />;
+
```

Comment:
[TODO] Refactor this into a new component that will be rendered at a specific route, and then redirect to the venue after successful login.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1523/commits/bba51569720b6b1215b83b06e481f811d677ed52

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1523/commits/bba51569720b6b1215b83b06e481f811d677ed52